Exception in thread "main" java.lang.NumberFormatException: For input string: "AboveAverage.cs"

### Expected behavior
generate report


### Actual behavior
This error happens in `web.log`:
```
2019.09.28 00:03:57 ERROR web[][o.s.s.p.w.RootFilter] Processing of request /api/cnesreport/report?key=aspose.cells.0827&author=Administrator&token=d17e5de54d26dda886e4b4fed6fe2248670aa0d7&generation=Generate failed
javax.servlet.ServletException: Filter execution threw an exception
at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:200)
at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
at org.sonar.server.user.UserSessionFilter.doFilter(UserSessionFilter.java:87)
at org.sonar.server.user.UserSessionFilter.doFilter(UserSessionFilter.java:71)
at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
at org.sonar.server.platform.web.CacheControlFilter.doFilter(CacheControlFilter.java:76)
at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
at org.sonar.server.platform.web.SecurityServletFilter.doHttpFilter(SecurityServletFilter.java:76)
at org.sonar.server.platform.web.SecurityServletFilter.doFilter(SecurityServletFilter.java:48)
at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
at org.sonar.server.platform.web.RedirectFilter.doFilter(RedirectFilter.java:58)
at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
at org.sonar.server.platform.web.requestid.RequestIdFilter.doFilter(RequestIdFilter.java:63)
at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
at org.sonar.server.platform.web.RootFilter.doFilter(RootFilter.java:62)
at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
at org.apache.catalina.filters.SetCharacterEncodingFilter.doFilter(SetCharacterEncodingFilter.java:109)
at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:199)
at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:493)
at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:140)
at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81)
at ch.qos.logback.access.tomcat.LogbackValve.invoke(LogbackValve.java:256)
at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87)
at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:342)
at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:800)
at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:806)
at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1498)
at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
at java.base/java.lang.Thread.run(Thread.java:835)
Caused by: java.lang.NoClassDefFoundError: org/apache/poi/openxml4j/exceptions/OpenXML4JException
at fr.cnes.sonar.plugin.ws.ExportTask.handle(ExportTask.java:78)
at org.sonar.server.ws.WebServiceEngine.execute(WebServiceEngine.java:110)
at org.sonar.server.ws.WebServiceFilter.doFilter(WebServiceFilter.java:88)
at org.sonar.server.platform.web.MasterServletFilter$GodFilterChain.doFilter(MasterServletFilter.java:126)
at org.sonar.server.platform.web.MasterServletFilter.doFilter(MasterServletFilter.java:95)
at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
... 41 common frames omitted
Caused by: java.lang.ClassNotFoundException: org.apache.poi.openxml4j.exceptions.OpenXML4JException
at org.sonar.classloader.SelfFirstStrategy.loadClass(SelfFirstStrategy.java:41)
at org.sonar.classloader.ClassRealm.loadClass(ClassRealm.java:87)
at org.sonar.classloader.ClassRealm.loadClass(ClassRealm.java:76)
... 47 common frames omitted
```

### Steps to reproduce behavior
```
java -jar sonar-cnes-report-3.1.0.jar -t 34ff2992bfa6f79a5118fa6297c15a8c514c23d4 -s http://192.168.1.240:9000 -p  xxxx
```

```
SonarQube URL: http://192.168.1.240:9000
SonarQube online: true
Detected SonarQube version: 7.9.1
Exception in thread "main" java.lang.NumberFormatException: For input string: "AboveAverage.cs"
        at sun.misc.FloatingDecimal.readJavaFormatString(Unknown Source)
        at sun.misc.FloatingDecimal.parseDouble(Unknown Source)
        at java.lang.Double.parseDouble(Unknown Source)
        at java.lang.Double.valueOf(Unknown Source)
        at fr.cnes.sonar.report.providers.ComponentProvider.getMinMetric(ComponentProvider.java:87)
        at fr.cnes.sonar.report.providers.ComponentProvider.getMetricStats(ComponentProvider.java:125)
        at fr.cnes.sonar.report.factory.ReportModelFactory.create(ReportModelFactory.java:101)
        at fr.cnes.sonar.report.ReportCommandLine.execute(ReportCommandLine.java:134)
        at fr.cnes.sonar.report.ReportCommandLine.main(ReportCommandLine.java:77)
```

### Detection version
**cnesreport version:** 3.1.0
**SonarQube version:** 7.9.1

--- a/src/main/java/fr/cnes/sonar/report/providers/ComponentProvider.java
+++ b/src/main/java/fr/cnes/sonar/report/providers/ComponentProvider.java
@@ -27,8 +27,10 @@ import org.apache.commons.lang3.math.NumberUtils;
 
 import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
 
 public class ComponentProvider extends AbstractDataProvider {
     /**
@@ -40,10 +42,15 @@ public class ComponentProvider extends AbstractDataProvider {
      * @param project The branch of the component to report.
      */
     ArrayList<Map> componentsList;
+    final Set<String> excludeMetricSet;
+
     public ComponentProvider(final SonarQubeServer server, final String token, final String project,
             final String branch) {
         super(server, token, project, branch);
         componentsList =  new ArrayList<>();
+        excludeMetricSet = new HashSet<>();
+        excludeMetricSet.add("Name");
+        excludeMetricSet.add("Path");
     }
 
     /**
@@ -133,6 +140,11 @@ public class ComponentProvider extends AbstractDataProvider {
 
 
     private boolean isCountableMetric(String metric){
+        // The metric is to be excluded
+        if(excludeMetricSet.contains(metric)){
+            return false;
+        }
+
         // Get first non-null value of metrics
         int i=0;
         while(i<componentsList.size() && (componentsList.get(i) == null || componentsList.get(i).get(metric) == null)){

